masp specs modifications (#152)
anoma/namada-docs: packages/specs/pages/masp/shielded-transfer.mdx

# Shielded Transfer

## Introduction and approach
To integrate a sapling-like circuit (as MASP) into Namada we have to take the differences between the execution models
between Namada and Zcash, for which sapling was originally specified.
AS Zcash does not follow exactly a UTXO model, due to the presence of Notes in architecture. However, the spending of a
note in Zcash is designed to minimize the distance between a pure UTXO model and a note-based architecture.
Namada, on the other and, uses an account-state model for execution. The approach used is to embed a Zcash transaction (as
specified within the zcash specification) in a Namada transfer object.

## Transfer and Transaction

> Here we are going to describe how we embed transactions as described in zcash, into Namada-compatible transfer objects.

Reviewer: ```suggestion
This section describes how to embed transactions as described in zcash, into Namada-compatible transfer objects.
```
